pandas/core/base.py

```diff
@@ -768,8 +769,21 @@ def base(self):
         return self.values.base
 
     @property
-    def _values(self):
-        """ the internal implementation """
+    def _ndarray_values(self):
+        """The data as an ndarray, possibly losing information.
+
+        The expectation is that this is cheap to compute.
+
+        - categorical -> codes
+
+        See '_values' for more.
+        """
+        # type: () -> np.ndarray
+        from pandas.core.dtypes.common import is_categorical_dtype
+
+        if is_categorical_dtype(self):
```

Comment:
why do we have this at all, e.g. ``np.array(ea)`` should just do the right thing no? why are we adding something else? note I actually don't mind that we have an additional property that we use consistently, more to the point of why ``__array__`` does not just return ``_ndarray_values``